### PR TITLE
PIL-1210-Terms of Service UI

### PR DESCRIPTION
**Pull request by divyaBacancy**
https://user-images.githubusercontent.com/66996635/142193773-bb8f0e97-bdac-4389-8214-c9bc7c7402b9.mov

https://user-images.githubusercontent.com/66996635/142196322-279cb30d-19c4-48c9-ab90-e781b0afab9e.mov

### CODE REVIEW

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few minor things in regards to our earlier discussion on splitting out the screens - cheers!

File: src/screens/NewProfile/NewProfile.js [hunk 1]
```diff
@@ -30,12 +30,10 @@ import { BaseText, MediumText, Paragraph } from 'components/legacy/Typography';
 import Button from 'components/legacy/Button';
 import ProfileImage from 'components/ProfileImage';
 import Checkbox from 'components/legacy/Checkbox';
-import PrismicDocumentModal from 'components/Modals/PrismicDocumentModal';
 import TextInput from 'components/legacy/TextInput';
-import Modal from 'components/Modal';
 
 // Constants
-import { PERMISSIONS, SET_WALLET_PIN_CODE } from 'constants/navigationConstants';
+import { PERMISSIONS, SET_WALLET_PIN_CODE, TERMS_AND_PRIVACY } from 'constants/navigationConstants';
```
Comment (IAmKio) [inline src/screens/NewProfile/NewProfile.js:36]: Could we change this to `LEGAL_MODAL` or something similar for consistency with the rest of the naming please?

**Comment by divyaBacancy** [comment]
Done with the PR comments. 

https://user-images.githubusercontent.com/66996635/143430269-7bd90778-773e-4b27-aecb-f597b8cf9574.mov

File: src/screens/Menu/Menu/components/MenuFooter.js [hunk 2]
```diff
@@ -27,11 +27,13 @@ import { useTranslationWithPrefix } from 'translations/translate';
...
 import Button from 'components/core/Button';
-import Modal from 'components/Modal';
-import PrismicDocumentModal from 'components/Modals/PrismicDocumentModal';
 
 // Constants
-import { MENU_SYSTEM_INFORMATION, BACKUP_WALLET_IN_SETTINGS_FLOW } from 'constants/navigationConstants';
+import {
+  MENU_SYSTEM_INFORMATION,
+  BACKUP_WALLET_IN_SETTINGS_FLOW,
+  TERMS_AND_PRIVACY,
```
Comment (IAmKio) [inline src/screens/Menu/Menu/components/MenuFooter.js:35]: Could we change this to LEGAL_MODAL or something similar for consistency with the rest of the naming please?

File: src/navigation/appNavigation.js [hunk 3]
```diff
@@ -556,6 +558,7 @@ const AppFlowNavigation = createStackNavigator(
     [ENS_MIGRATION_FLOW]: ensMigrationFlow,
     [ADD_CASH]: addCashFlow,
     [WEB_VIEW]: WebViewScreen,
+    [TERMS_AND_PRIVACY]: TermsAndPrivacyScreen,
```
Comment (IAmKio) [inline src/navigation/appNavigation.js:561]: Could we change this to LEGAL_MODAL or something similar for consistency with the rest of the naming please?

**Review by IAmKio — CHANGES_REQUESTED** [review]
3 comments but they're all the name, just the renaming of a const for consistency. Thanks!

**Review by IAmKio — APPROVED** [review]

**Pull request merged**